ansible/roles/apache/templates/etc/pki/hooks/apache2.j2

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2014-2017 Maciej Delmanowski <drybjed@drybjed.net>
+# Copyright (C) 2015-2017 Robin Schneider <ypid@riseup.net>
+# Copyright (C) 2014-2017 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Please update the copyright. See my other comment.

Reply:
Fixed